draft-ietf-scitt-architecture.md

```diff
@@ -271,7 +271,7 @@ The identity of a Transparency Service is captured by a public key that must be
 
 Transparent Statement:
 
-: a Signed Statement that is augmented with a Receipt created via Registration at a Transparency Services (stored in the unprotected header of COSE Envelope of the Signed Statement).
+: a Signed Statement that is augmented with a Receipt created via Registration in a Transparency Service(stored in the unprotected header of COSE Envelope of the Signed Statement).
```

Comment:
```suggestion
: a Signed Statement that is augmented with a Receipt created via Registration in a Transparency Service ( the receipt is stored under tag TBD_N in the unprotected header of the Signed Statement).
```

Reply:
I would prefer avoiding specific here. I will make it a general statement